app/src/converter.js

```diff
@@ -2,17 +2,52 @@ import open from 'open';
 
 const os = require("os");
 const path = require('path');
+const fs = require("node:fs");
 
 const logger = require('./logger');
 const rpc = require('./weh-rpc');
 
 const exec_dir = path.dirname(process.execPath);
-let ffmpeg = path.join(exec_dir, "ffmpeg");
-let ffprobe = path.join(exec_dir, "ffprobe");
 
-if (os.platform() == "win32") {
-  ffmpeg += ".exe";
-  ffprobe += ".exe";
+const use_prebuilt_ffmpeg = fileExistsSync(ensureProgramExt(path.join(exec_dir, "ffmpeg")));
```

Comment:
This is only used for test. We'll find a different way. Remove.

Reply:
OK Removed.